transactions/src/main/java/org/jboss/as/txn/service/ArjunaRecoveryManagerService.java

```diff
@@ -26,20 +26,25 @@
 import java.util.Collections;
 import java.util.List;
 
+import com.arjuna.ats.arjuna.recovery.RecoveryManager;
 import com.arjuna.ats.internal.jta.recovery.jts.JCAServerTransactionRecoveryModule;
 import org.jboss.as.network.ManagedBinding;
 import org.jboss.as.network.SocketBinding;
 import org.jboss.as.network.SocketBindingManager;
 import org.jboss.as.server.suspend.SuspendController;
 import org.jboss.as.txn.logging.TransactionLogger;
 import org.jboss.as.txn.suspend.RecoverySuspendController;
+import org.jboss.jbossts.xts.initialisation.RecoveryInitialisation;
+import org.jboss.jbossts.xts.recovery.participant.ba.XTSBARecoveryManager;
```

Comment:
Can't we put something in the object store record? I do not think we should couple transactions subsystem to web-services.